src/Compiler/CMakeLists.txt

```diff
@@ -68,6 +65,7 @@ add_onnx_mlir_library(OMCompilerPasses
   LINK_LIBS PUBLIC
   ${OMLibs}
   OMCompilerOptions
+  MLIRIR
```

Comment:
Nit: Generally the link libs try to be alphabetic. Below as well.

Reply:
Done